hack/update-bazel.sh

```diff
@@ -64,12 +64,16 @@ done
 # filter out known pkg-config error (see buildozer workaround below)
 # NOTE: the `|| exit "${PIPESTATUS[0]}"` is to ignore grep errors 
 # while preserving the exit code of gazelle
+KNOWN_PKG_CONFIG_ERROR_PACKAGES=(
```

Comment:
we should add a warning to be careful when ignoring these. in the case of libseccomp we actually needed to modify the script below to handle it

Reply:
How about adding some comments like this? @BenTheElder 
```
Warning: Please be careful to ignore these pkg-config errors. 
As gazelle doesn't support pkg-config, if we suppress these errors here, we actually need to modify the script below to handle it manually.
For instance, in the case of `libseccomp`, we set the link option `-lseccomp` to make it work properly.
```

Reply:
> Warning: Please be careful to ignore these pkg-config errors. 

I think this should read:
> Warning: Please be careful when ignoring these pkg-config errors.

Reply:
thanks!

Reply:
@BenTheElder 
I have updated the code. Thanks for your help! :)